hack/verify-lws-controller-manifests.sh

```diff
@@ -32,4 +32,9 @@ rm -rf "${TMP_DIR}"
 if [ -n "$(git status --porcelain -- bindata/assets/lws-controller-generated/)" ];then
```

Comment:
```
pushd "${SCRIPT_ROOT}"
```

it is good to ensure we are in our directory before we call git

Reply:
Sounds good. Updated.